src/components/DateRangePicker.js

```diff
@@ -447,13 +447,14 @@ const DateRangePicker = React.createClass({
         marginTop: isLargeOrMediumWindowSize ? 10 : 5,
         position: 'absolute',
         left: this.props.isRelative ? 'auto' : 0,
-        right: isLargeOrMediumWindowSize ? 0 : 'auto',
+        right: 0,
```

Comment:
Really no matter what we want the calendar anchored to the right. On very small devices anchoring to the right 0 and left 0 allows the 100% width effect without the flex wrap breaking.